pkg/blobstore/grpcclients/cas_zstd_blob_access.go

```diff
@@ -0,0 +1,265 @@
+package grpcclients
```

Comment:
I don't see why this needs to be a separate backend. Please integrate this functionality into the existing `cas_blob_access.go`.